core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -10,6 +10,14 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
 
     "transpile queries" in {
 
+      /* Test BANG ! command */
```

Comment:
create new `should` block right above `transpile queries` in approx the following way:

```
"snowsql commands" should {
   "transpile BANG" in {
      // ... move assertions here
   }
}
```

remove redundant `/* Test BANG ! command */` comments

Reply:
done